hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/RestoreTablesClient.java

```diff
@@ -69,6 +69,12 @@ public RestoreTablesClient(Connection conn, RestoreRequest request) {
     this.isOverwrite = request.isOverwrite();
     this.conn = conn;
     this.conf = conn.getConfiguration();
+    if (request.getTargetRootDir() != null) {
```

Comment:
I wonder if we should rename this to getRestoreRootDir... just so its symmetrical with getBackupRootDir

Reply:
Yeah that's a good idea. I'll update.